dbt_semantic_interfaces/protocols/metric.py

```diff
@@ -46,13 +46,13 @@ def post_aggregation_measure_reference(self) -> MeasureReference:
 
     @property
     @abstractmethod
-    def join_to_timespine(self) -> bool:
-        """If the measure should be joined to the timespine."""
+    def time_spine_join(self) -> bool:
+        """If the measure should be joined to the time spine."""
         pass
 
     @property
     @abstractmethod
-    def fill_nulls_with(self) -> Optional[int]:
+    def null_fill_value(self) -> Optional[float]:
```

Comment:
These are hard breaking changes on the dbt-core parser so we can't push these into dbt-semantic-interfaces or take dependencies on these names in MetricFlow if we want to be able to backport things.

It's fine to add duplicate fields with the new names, and once we get support for both broadly available across the releases we are committed to supporting we can gradually remove the old ones, but given that 1) these names were chosen within the context of existing dbt metrics nomenclature and 2) they're now out in the world I'm not sure this adjustment is worth the effort.